### PR TITLE
functions duplicate checks added (#29)

### LINKED ISSUE #29: Some functions don't have duplicate checks
`twentytwentytwo_scripts` and `twentytwentytwo_editor_styles` functions don't have duplicate checks with `function_exists` function.
I'll love to send PR after hearing from a maintainer... :)

### PR DESCRIPTION
**Pull request by AbmSourav**
Fixes #29 
Function duplicate checks added for `twentytwentytwo_support` and `twentytwentytwo_scripts`

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -15,24 +15,28 @@ function twentytwentytwo_support()  {
...
+if ( ! function_exists( 'twentytwentytwo_scripts' ) ) :
+	/**
+	 * Enqueue scripts and styles.
+	 */
+	function twentytwentytwo_scripts() {
+		// Enqueue theme stylesheet.
+		wp_enqueue_style( 'twentytwentytwo-style', get_template_directory_uri() . '/style.css', array(), wp_get_theme()->get( 'Version' ) );
+	}
+	add_action( 'wp_enqueue_scripts', 'twentytwentytwo_scripts' );
+endif;
```
Comment (pattonwebz) [inline functions.php]: Would this function be better named as `twentytwentytwo_styles`?

Comment (AbmSourav) [reply functions.php]: @pattonwebz That can be another issue, this PR is already following a specific issue #29

Comment (pattonwebz) [reply functions.php]: Agreed, but wanted to flag :)

**Comment by AbmSourav** [comment]
Hi @pattonwebz, Thanks for reviewing my codes. 👍🏼

**Review by pattonwebz — APPROVED** [review]
These changes look good to me. Thanks for the PR :)

**Review by kjellr — APPROVED** [review]
They look good to me too. I agree we can change this to `twentytwentytwo_styles` — we don't plan on adding any scripts anyway. Anyone should feel free to do that in a followup PR!

**Pull request merged**